### PR TITLE
Bumped Heapster to v1.1.0-beta2

### PR DESCRIPTION
**Pull request by piosz**
cc @a-robinson @fgrzadkowski @jszczepkowski @Q-Lee

### CODE REVIEW

**Comment by mwielgus** [comment]
LGTM

File: cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml [hunk 1]
```diff
@@ -40,7 +40,6 @@ spec:
           command:
             - /heapster
             - --source=kubernetes.summary_api:''
-            - --metric_resolution=60s
```
Comment (a-robinson) [inline cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml]: Just so I understand, why is this flag being removed?

**Comment by a-robinson** [comment]
LGTM, just one question to help me understand the new config

File: cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml (hunk 1 above)
Comment (piosz) [inline cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml]: I changed it to be 60s by default in Heapster https://github.com/kubernetes/heapster/pull/1164.
There is not much sense to have it different and overwrite in config.

Comment (a-robinson) [inline cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml]: Ack, thanks!

**Comment by Q-Lee** [comment]
lgtm

**Comment by piosz** [comment]
Actually bumped the Heapster. Previously I only changed versions not the image.

**Comment by piosz** [comment]
@k8s-bot unit test this please

**Comment by piosz** [comment]
flake due to #25897

**Pull request merged**